freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -95,7 +95,11 @@ def handle(self, event):
         )
 
         try:
-            return self._handle_or_skip(event)
+            builds = self._handle_or_skip(event)
+            self.start_to_build_images(builds.values())
+            msg = 'Rebuilding %d container images.' % (len(builds))
+            db_event.transition(EventState.BUILDING, msg)
+            return builds
```

Comment:
Better to move this outside the try..except block for clarity (`SkipEventException` is only thrown in `_handle_or_skip()`).

Reply:
Ack. Fixed